packages/shylu/shylu_dd/frosch/src/SchwarzPreconditioners/FROSch_TwoLevelPreconditioner_decl.hpp

```diff
@@ -108,6 +108,7 @@ namespace FROSch {
         
         std::string description() const;
         
+        int resetMatrix(CrsMatrixPtr &k);
```

Comment:
When do we need this?

Reply:
atm this is still needed for a reset of the underlying matrix. -> Added to ToDo-List